### ISSUE TITLE
运行panopticdeeplab时候环境都搭好了，运行时直接进程结束了

### ISSUE THREAD

**Issue by Invisible2733** [PaddlePaddle/PaddleSeg#2025]
small_instance_weight: 3
  transforms:
  - label_padding_value:
    - 0
    - 0
    - 0
    target_size:
    - 2049
    - 1025
    type: Padding
  - type: Normalize
  type: CityscapesPanoptic
------------------------------------------------
W0418 17:33:10.465845 13064 device_context.cc:447] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 11.1, Runtime API Version: 11.1
W0418 17:33:10.466815 13064 device_context.cc:465] device: 0, cuDNN Version: 8.1.
2022-04-18 17:33:15 [INFO]	Loading pretrained model from https://bj.bcebos.com/paddleseg/dygraph/resnet50_vd_ssld_v2.tar.gz
2022-04-18 17:33:16 [INFO]	There are 275/275 variables loaded into ResNet_vd.
2022-04-18 17:33:16 [WARNING]	`learning_rate` in configuration file will be deprecated, please use `lr_scheduler` instead. E.g
            lr_scheduler:
                type: PolynomialDecay
                learning_rate: 0.01
D:\anaconda3\envs\pp\lib\site-packages\paddle\nn\layer\norm.py:653: UserWarning: When training, we now always track global mean and variance.
  "When training, we now always track global mean and variance.")

进程已结束，退出代码 -1073740791 (0xC0000409)

欢迎您反馈PaddleSeg使用问题，辛苦您提供以下信息，方便我们快速定位和解决问题：
 1. PaddleSeg版本：（请提供版本号和分支信息，如PaddleSeg release/2.3）
 2. PaddlePaddle版本：（如PaddlePaddle 2.1.0）
 3. 操作系统信息：win10
 4. Python版本号：py3.7
 5. CUDA/cuDNN版本：cuda11.1 cudnn8.1
 6. 其他内容: （增加其他与问题相关的内容）

**Comment by juncaipeng** [comment]
安装过程中验证执行正确吗？quick_start中demo执行正确吗？
麻烦提供你详细的yaml文件和执行命令。

**Comment by Invisible2733** [comment]
**验证执行正确的：**
![image](https://user-images.githubusercontent.com/70048885/163819447-e828b9c7-7976-42f3-8bcd-e2e49c8ccd68.png)
**请问quick_start中demo是指？**
yml文件采用的是panopticdeeplab\configs下的：
![image](https://user-images.githubusercontent.com/70048885/163820100-171e3953-c6af-4bfd-a567-8df7a04b0363.png)
**执行命令是直接运行train.py文件，以下是运行结果：**
D:\anaconda3\envs\pp\python.exe C:/Users/Lenovo/PaddleSeg/contrib/PanopticDeepLab/train.py --config=C:\Users\Lenovo\PaddleSeg\contrib\PanopticDeepLab\configs\panoptic_deeplab\panoptic_deeplab_resnet50_os32_cityscapes_1025x513_bs8_90k_lr00005.yml
D:\anaconda3\envs\pp\lib\site-packages\paddle\vision\transforms\functional_pil.py:36: DeprecationWarning: NEAREST is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.NEAREST or Dither.NONE instead.
  'nearest': Image.NEAREST,
D:\anaconda3\envs\pp\lib\site-packages\paddle\vision\transforms\functional_pil.py:37: DeprecationWarning: BILINEAR is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.BILINEAR instead.
  'bilinear': Image.BILINEAR,
D:\anaconda3\envs\pp\lib\site-packages\paddle\vision\transforms\functional_pil.py:38: DeprecationWarning: BICUBIC is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.BICUBIC instead.
  'bicubic': Image.BICUBIC,
D:\anaconda3\envs\pp\lib\site-packages\paddle\vision\transforms\functional_pil.py:39: DeprecationWarning: BOX is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.BOX instead.
  'box': Image.BOX,
D:\anaconda3\envs\pp\lib\site-packages\paddle\vision\transforms\functional_pil.py:40: DeprecationWarning: LANCZOS is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.LANCZOS instead.
  'lanczos': Image.LANCZOS,
D:\anaconda3\envs\pp\lib\site-packages\paddle\vision\transforms\functional_pil.py:41: DeprecationWarning: HAMMING is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.HAMMING instead.
  'hamming': Image.HAMMING
2022-04-18 22:04:30 [INFO]	
------------Environment Information-------------
platform: Windows-10-10.0.19041-SP0
Python: 3.7.13 (default, Mar 28 2022, 08:03:21) [MSC v.1916 64 bit (AMD64)]
Paddle compiled with cuda: True
NVCC: Build cuda_11.1.relgpu_drvr455TC455_06.29069683_0
cudnn: 8.1
GPUs used: 1
CUDA_VISIBLE_DEVICES: None
GPU: ['GPU 0: GeForce MX150']
PaddleSeg: 2.4.0
PaddlePaddle: 2.2.2
OpenCV: 4.5.5
------------------------------------------------
2022-04-18 22:04:32 [INFO]	
---------------Config Information---------------
batch_size: 8
iters: 90000
learning_rate:
  decay:
    end_lr: 0.0
    power: 0.9
    type: poly
  value: 5.0e-05
loss:
  coef:
  - 1
  - 200
  - 0.001
  types:
  - ignore_index: 255
    top_k_percent_pixels: 0.2
    type: CrossEntropyLoss
  - ignore_index: 255
    reduction: none
    type: MSELoss
  - ignore_index: 255
    reduction: none
    type: L1Loss
model:
  align_corners: true
  aspp_out_channels: 256
  aspp_ratios:
  - 1
  - 3
  - 6
  - 9
  backbone:
    output_stride: 32
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/resnet50_vd_ssld_v2.tar.gz
    type: ResNet50_vd
  backbone_indices:
  - 2
  - 1
  - 0
  - 3
  decoder_channels: 256
  instance_aspp_out_channels: 256
  instance_class_key:
  - center
  - offset
  instance_decoder_channels: 128
  instance_head_channels: 32
  instance_low_level_channels_projects:
  - 64
  - 32
  - 16
  instance_num_classes:
  - 1
  - 2
  low_level_channels_projects:
  - 128
  - 64
  - 32
  type: PanopticDeepLab
optimizer:
  type: adam
train_dataset:
  dataset_root: data/cityscapes
  ignore_stuff_in_offset: true
  mode: train
  small_instance_area: 4096
  small_instance_weight: 3
  transforms:
  - max_scale_factor: 2.0
    min_scale_factor: 0.5
    scale_step_size: 0.25
    type: ResizeStepScaling
  - crop_size:
    - 1025
    - 513
    label_padding_value:
    - 0
    - 0
    - 0
    type: RandomPaddingCrop
  - type: RandomHorizontalFlip
  - brightness_range: 0.4
    contrast_range: 0.4
    saturation_range: 0.4
    type: RandomDistort
  - type: Normalize
  type: CityscapesPanoptic
val_dataset:
  dataset_root: data/cityscapes
  ignore_stuff_in_offset: true
  mode: val
  small_instance_area: 4096
  small_instance_weight: 3
  transforms:
  - label_padding_value:
    - 0
    - 0
    - 0
    target_size:
    - 2049
    - 1025
    type: Padding
  - type: Normalize
  type: CityscapesPanoptic
------------------------------------------------
W0418 22:04:34.615145 11812 device_context.cc:447] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 11.1, Runtime API Version: 11.1
W0418 22:04:34.615145 11812 device_context.cc:465] device: 0, cuDNN Version: 8.1.
2022-04-18 22:04:36 [INFO]	Loading pretrained model from https://bj.bcebos.com/paddleseg/dygraph/resnet50_vd_ssld_v2.tar.gz
2022-04-18 22:04:38 [INFO]	There are 275/275 variables loaded into ResNet_vd.
2022-04-18 22:04:38 [WARNING]	`learning_rate` in configuration file will be deprecated, please use `lr_scheduler` instead. E.g
            lr_scheduler:
                type: PolynomialDecay
                learning_rate: 0.01
D:\anaconda3\envs\pp\lib\site-packages\paddle\nn\layer\norm.py:653: UserWarning: When training, we now always track global mean and variance.
  "When training, we now always track global mean and variance.")

进程已结束，退出代码 -1073740791 (0xC0000409)

**Comment by Invisible2733** [comment]
执行readme中的python -m paddle.distributed.launch train.py \
       --config configs/panoptic_deeplab/panoptic_deeplab_resnet50_os32_cityscapes_1025x513_bs8_90k_lr00005.yml \
       --do_eval \
       --use_vdl \
       --save_interval 5000 \
       --save_dir output
也是出现同样的问题。
